src/runtime_net.cpp

```
@@ -137,6 +137,10 @@ bool __ry_is_private_addrinfo(const struct addrinfo *info) {
     return ry_net_is_private_addrinfo(info);
 }
 
+bool __ry_is_private_addr(const struct sockaddr *sa) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)